src/main/java/com/example/bookshop/service/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package com.example.bookshop.service;
+
+import com.example.bookshop.dto.UserLoginRequestDto;
+import com.example.bookshop.dto.UserLoginResponseDto;
+import com.example.bookshop.security.JwtUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Autowired
+    private JwtUtil jwtUtil;
```

Comment:
Try to inject by constructor and you can use @requiredArgsConstructor of lombok for this but fields should be final